code/modules/reagents/chemistry/reagents/water.dm

```diff
@@ -364,15 +364,10 @@
 /datum/reagent/fuel/unholywater/on_mob_life(mob/living/M)
 	var/update_flags = STATUS_UPDATE_NONE
 	if(iscultist(M))
-		M.AdjustDrowsy(-10 SECONDS)
-		M.AdjustParalysis(-2 SECONDS)
-		M.AdjustStunned(-4 SECONDS)
-		M.AdjustWeakened(-4 SECONDS)
-		M.AdjustKnockDown(-4 SECONDS)
-		update_flags |= M.adjustStaminaLoss(-25, FALSE)
```

Comment:
Personally, I'd reduce the stun times to 2 cycles for everything, leave drowsy alone, and reduce the stamina heal to 10.

I dunno if going to the extreme of completely removing the functionality is necessarily a good thing.

Reply:
I buffed the healing factor on the stats, and returned its drowsy buff to beat ether cheese, however im not interested in returning the stun resistances due to what ive said before. Cult already has many tools for fighting disabler/batons and it doesn't need more.